react/react1/week3/vite-project/src/todo-list-item.jsx

```diff
@@ -0,0 +1,53 @@
+import { Link } from "react-router-dom";
+import "./index.css";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const ListItem = ({ todo, onDelete }) => {
+  const [checked, setChecked] = useState(todo.completed);
```

Comment:
We can make this work without this state by having a function prop that changes the competed value on the parent component